### PR TITLE
No-post subject line flag

### PR DESCRIPTION
**Pull request by bion**
https://www.pivotaltracker.com/story/show/98505258

### CODE REVIEW

File: src/core/prism.coffee [hunk 1]
```diff
@@ -28,8 +28,7 @@ class Prism
         wordpress.massageThreadMaps()
         debug "sending threads to wp: ", pjson(threadId for threadId of threads)
         for threadId, thread of threads
-          do (threadId, thread) =>
-            setTimeout (=> wordpress.writeThread thread), delay * postCount++
+          setTimeout (=> wordpress.writeThread thread), delay * postCount++
```
Comment (dukedorje) [inline src/core/prism.coffee]: pretty sure we added the do {} when it didn't work.  i think thread gets rewritten for the closure to the last iteration of the loop, and when the callback is called they are all the same value

Comment (bion) [inline src/core/prism.coffee]:
```javascript
```

```javascript
            for (threadId in threads) {
              thread = threads[threadId];
              results1.push(setTimeout((function() {
                return wordpress.writeThread(thread);
              }), delay * postCount++));
            }
```

Comment (bion) [inline src/core/prism.coffee]: yep, didn't think it worked that way but it totally does. thanks, @dukedorje

Comment (bion) [inline src/core/prism.coffee]: tested with:
```coffeescript
nums =
  one: 1
  two: 2
  three: 3

for numName, num of nums
  setTimeout (=> console.log(num)), 100
```

**Comment by dukedorje** [comment]
:+1:

**Comment by dukedorje** [comment]
:shipit:

**Pull request merged**